docs/src/modules/components/ApiPage.tsx

```diff
@@ -360,6 +360,26 @@ export default function ApiPage(props: ApiPageProps) {
           layoutStorageKey={layoutStorageKey.classes}
           displayClassKeys
         />
+
+        {pageContent.filename ? (
+          <div
+            className="MuiCallout-root MuiCallout-info"
+            style={{
+              alignItems: 'baseline',
+              gap: '4px',
+              marginTop: 0,
+            }}
+          >
+            <p
```

Comment:
If we use the callout, I guess we should use the new design:

```suggestion
            <div className=MuiCallout-icon-container" />
            <p className="MuiCallout-content"
```